release/rllib_tests/learning_tests/yaml_files/cql/cql-halfcheetahbulletenv-v0.yaml

```diff
@@ -25,19 +28,18 @@ cql-halfcheetahbulletenv-v0:
         no_done_at_end: false
         n_step: 3
         rollout_fragment_length: 1
-        replay_buffer_config:
-          type: MultiAgentReplayBuffer
-          learning_starts: 256
+        num_workers: 8
+        grad_clip: 40
         train_batch_size: 256
         target_network_update_freq: 0
         min_train_timesteps_per_iteration: 1000
         optimization:
             actor_learning_rate: 0.0001
             critic_learning_rate: 0.0003
             entropy_learning_rate: 0.0001
-        num_workers: 0
         num_gpus: 1
         metrics_smoothing_episodes: 5
+        min_time_s_per_iteration: 30
```

Comment:
Was this necessary? This is offline RL so more number of iterations should have worked equally here. If this is necessary something does not make sense, if it wasn't necessary we should remove this hparam and instead increase timesteps_total to be consistent with other learning tests.

Reply:
wait this parameter controls the logging frequency. I increased it so that we don't run too many unnecessary evaluations

Reply:
it's not just that, but also the training time spent per iteration. so you keep running the same training_step() function until that timing requirement is met. In other words you keep taking gradient updates until that time is reached.

Reply:
We chatted offline and my concern was more about reproducibility. These nits are not merge blockers. So please merge if it's the only thing holding this back.

Reply:
will open a separate pr that addresses this issue across all release tests and environments